templateRunScripts/run.icon

```diff
@@ -113,8 +112,8 @@
  if ( ! -d "$OUTDIR" ) mkdir -p $OUTDIR
 
  ls -l $BLD/$EXEC; size $BLD/$EXEC
- unlimit
- limit
+# unlimit
+# limit
```

Comment:
do we know what these odd `unlimit` and `limit` calls are meant to do? If not, maybe we just delete them and let git do the remembering weird obsolete hacks?

Reply:
It's to remove the mem/file handle limits for the current process and subprocesses. It's often sprinkled throughout scripts that are run on NCI. I'm not sure if it is actually relevant or purely defensive. 

I think that this can be moved to the slurm scripts that are used to run as its very nci specific

Reply:
I'll fix this in the next PR